README.md

```diff
@@ -1,48 +1,66 @@
 # Schematic
-[![Build Status](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Factions-badge.atrox.dev%2FSage-Bionetworks%2Fschematic%2Fbadge%3Fref%3Ddevelop&style=flat)](https://actions-badge.atrox.dev/Sage-Bionetworks/schematic/goto?ref=develop) [![Documentation Status](https://readthedocs.org/projects/sage-schematic/badge/?version=develop)](https://sage-schematic.readthedocs.io/en/develop/?badge=develop)
-
-
-- [Schematic](#schematic)
-  - [Introduction](#introduction)
-  - [Installation Requirements and Pre-requisites](#installation-requirements-and-pre-requisites)
-  - [Package Setup Instructions](#package-setup-instructions)
-  - [Command Line Interface](#command-line-interface)
-  - [Contributing](#contributing)
-  - [Contributors](#contributors)
+[![Build Status](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Factions-badge.atrox.dev%2FSage-Bionetworks%2Fschematic%2Fbadge%3Fref%3Ddevelop&style=flat)](https://actions-badge.atrox.dev/Sage-Bionetworks/schematic/goto?ref=develop) [![Documentation Status](https://readthedocs.org/projects/sage-schematic/badge/?version=develop)](https://sage-schematic.readthedocs.io/en/develop/?badge=develop) [![PyPI version](https://badge.fury.io/py/schematicpy.svg)](https://badge.fury.io/py/schematicpy)
 
 ## Introduction
 
-SCHEMATIC is an acronym for _Schema Engine for Manifest Ingress and Curation_. The Python based infrastructure provides a _novel_ schema-based, data ingress ecosystem, that is meant to streamline the process of metadata annotation and validation for various data contributors.
+SCHEMATIC is an acronym for _Schema Engine for Manifest Ingress and Curation_. The Python based infrastructure provides a _novel_ schema-based, data ingress ecosystem, that is meant to streamline the process of dataset annotation, metadata validation and submission to an asset store for various data contributors.
 
 ## Installation Requirements and Pre-requisites
 
 * Python 3.7.1 or higher
-* [`pyenv`](https://github.com/pyenv/pyenv)
-* [`poetry`](https://github.com/python-poetry/poetry)
 
+Note: You need to be a registered and certified user on [`synapse.org`](https://www.synapse.org/), and also have the right permissions to download the Google credentials files from Synapse.
+
+## Installing
+
+Create and activate a virtual environment within which you can install the package:
```

Comment:
You probably need to confirm that users are running a recent enough version of Python. I normally would suggest `pyenv` for this, but I'm running into issues with it on macOS (and so it Matt in SysBio). Maybe link to Miniconda for now.

Reply:
So in the [Installation Requirements and Pre-requisites](https://sage-schematic.readthedocs.io/en/develop-docs-improve/README.html#installation-requirements-and-pre-requisites) section of the documentation, there's a recommendation for users to use `pyenv` for managing various versions of Python. Do you think we should recommend users to use Miniconda instead?

Reply:
Maybe add a note that if they're having issues on macOS, they can try Miniconda. Ultimately, I think `pyenv` is better for managing Python versions.

Reply:
Right, makes sense. Fixed this accordingly.